### ISSUE TITLE
[Design] .get_attribute() should return a Option if null

### ISSUE THREAD

**Issue by audioXD** [stevepryde/thirtyfour#33]
# Problem
Currently `.get_attribute()`  returns a `"null"` if the attribute doesn't exist. 

# Intended hehavior
I think it should return a `None` insted of `"null"`

# Reason
If someone had a `<div data-some="null">`, how would that someone detect the attribute `data-some` is defined.

# Ways to implement
 - Change the` .get_attribute()` function to return a `Option<String>` (**BREAKING CHANGE**),
 - Add a method called ` .get_attribute_opt()`  that returns  a `Option<String>`,

**Comment by stevepryde** [comment]
The first option (breaking change) seems like the right approach here

**Comment by stevepryde** [comment]
I assume the same should apply to `get_property()` and `get_css_property()`

**Comment by audioXD** [comment]
I agree with `get_property()`, but i don't know for `get_css_property()`...

**Comment by audioXD** [comment]
Wouldn't a CSS property always have a value, unless name is invalid?

**Comment by audioXD** [comment]
I've tested it and when running
```
c.get("https://google.si").await?;
println!("Value: {:?}", c.query(By::Css(r#"html"#)).wait().first().await?.get_css_property("background-colo").await?);
println!("Value: {:?}", c.query(By::Css(r#"html"#)).wait().first().await?.get_css_property("background-color").await?);
```

It returns
```
Value: ""
Value: "rgba(0, 0, 0, 0)"
```

It returns a ampty value if the  name is invalid

**Comment by stevepryde** [comment]
If you look at the code for get_attribute() you will see that if the value is not a string type then it converts it into a string. That should probably just return None instead (and Some if it is a string type).

**Comment by stevepryde** [comment]
The same logic is used for the other two methods which is why I suggested they would benefit from the same change

**Comment by audioXD** [comment]
The [standard](https://www.w3.org/TR/webdriver2/#get-element-css-value) for `get_css_property()` says that to return `""` in the else case, so making it a Option wouldn't do anything

**Comment by stevepryde** [comment]
Fair enough 👍

**Comment by stevepryde** [comment]
Fixed in 9f506dce06b0997989e4284e6cd6fd5e09e324bd
